### DISCUSSION TITLE
Launch Ente Auth via Quick Settings Shade

### DISCUSSION THREAD

**Discussion by iamgitcat** [ente-io/ente#1026] [Enhancements]
**Launch Auth via Quick Settings Shade**
When some app requires OTP code you need to launch Ente Auth, now it is possible only via launcher, so you exit the app itself, launch Ente Auth, copy code, than launch app again. It is slow and inconvenient.

**Alternative how Aegis does it**
Aegis puts a button of itself in Android Quick Settings shade, so when app requires OTP code you do not need to exit the app, you just pull Quick Settings shade, click Aegis, copy code, Aegis closes and you are back in app instantly, without exiting the app at all. Super quick!

**Additional context**
Quick Settings Shade is where brightness, volume, Wi-Fi switches and sliders located.

**Comment by Hudzen10** [comment]
+1 for feature request.

Until then, it's possible to achieve similar using Tasker (https://tasker.joaoapps.com/) to create a Quick Settings tile that launches Auth.ente.io  :

1.
Create a simple task with 'Launch App' as one of the steps 
(I have step 1. 'Close System Dialogs' to close the Quick Settings shade & step 2. 'Launch App' with the Auth app selected)

2.
In Tasker's Preferences (Open Tasker > 3 vertical dot 'kebab' menu top right > Preferences) under the 'ACTION' tab select the task you created as one of the three available  'Quick Settings Tasks' options and save/OK/tick you're way back out of the preferences screen and make sure the changes are saved in Tasker.

3.
In Android's Quick Settings editor find the tile for the Tasker Quick Settings task you just configured, should work as described above 👍
